src/demo/java/org/junitpioneer/jupiter/ExpectedToFailExtensionDemo.java

```diff
@@ -32,10 +32,23 @@ void doSomething() {
 		int actual = brokenMethod();
 		assertEquals(10, actual);
 	}
-	// end::expected_to_fail_message[]
 
 	private int brokenMethod() {
 		return 0;
 	}
+	// end::expected_to_fail_message[]
```

Comment:
I hope you agree with me moving this end marker and doing the same for my new example.

Reply:
I don't think the two are equivalent. Your example showcases a planned feature with no body (which could be an actual example of someone using this) but the previous example doesn't actually showcase what the example is talking about... The `brokenMethod` is not actually "broken", and seeing it **not** returning 10 doesn't help understanding. Does this make sense?

I'd move back the tag here but leave the one in your example where it is.

Reply:
Yes, that makes sense.